extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -439,6 +445,9 @@ export class AmpA4A extends AMP.BaseElement {
       }
     }
 
+    this.detectedFont_ =
```

Comment:
Malte suggested that the best place to detect the font would be in the onLayoutMeasure function. If you think it preferable, I can store the variable in a4a.win instead of directly on a4a.